### PR TITLE
MAUI-[Others]-Add the column selection in the UG.

### PR DESCRIPTION
**Pull request by sudarsanMS**
### Description

- Add the column selection in the UG.

### CODE REVIEW

File: MAUI/DataGrid/selection.md [hunk 1]
```diff
@@ -122,6 +122,34 @@ public partial class MainPage : ContentPage
 
 ![All the rows selected in .NET MAUI DataGrid](Images/selection/maui-datagrid-selectall.png)
 
+## Column selection
+The SfDataGrid provides the [SfDataGrid.AllowColumnSelection]() property to select columns when the `SelectionMode` is set to Multiple, the `SelectionUnit` is set to Cell, and the `NavigationMode` is set to Cell.
+
+By default, the `SortingGestureType` is set to `Tap` mode, so we select the column by double-clicking the header cell. We can also modify this behavior by changing the `SortingGestureType`.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:128]: The `SfDataGrid` includes the [SfDataGrid.AllowColumnSelection](...) property, which allows you to select columns by tapping the column header cell. This feature is enabled only when the `SelectionMode` is set to `Multiple`, the `SelectionUnit` is set to `Cell`, and the `NavigationMode` is set to `Cell`.

**Note:** When the `SortingGestureType` is set to `Tap`, column selection is applied only when double-tapping the column header cell. This ensures both sorting and column selection interactions can occur without conflicts.

File: MAUI/DataGrid/selection.md [hunk 2]
```diff
@@ -122,6 +122,34 @@ public partial class MainPage : ContentPage
...
+
+{% tabs %}
+{% highlight XAML %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid"
+                    SelectionMode="Multiple"
+                    NavigationMode="Cell"
+                    SelectionUnit="Cell"
+                    AllowColumnSelection="True"
+                    ColumnWidthMode="Auto"
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:137]: Remove `ColumnWidthMode` and set `AutoGenerateColumnsMode` to none.

File: MAUI/DataGrid/selection.md [hunk 3]
```diff
@@ -122,6 +122,34 @@ public partial class MainPage : ContentPage
...
+   <syncfusion:DataGridTextColumn MappingName="CustomerID"
+                 HeaderText="Customer ID" />
+   <syncfusion:DataGridTextColumn MappingName="ShipCountry "
+                 HeaderText="Ship Country " />
+</syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Pull To Refresh" src="Images\selection\maui_datagrid_column_selection.gif" width="404"/>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/selection.md:151]: Column selection support in .NET MAUI DataGrid

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

File: MAUI/DataGrid/selection.md (hunk 1 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/selection.md:128]: modified

File: MAUI/DataGrid/selection.md (hunk 3 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/selection.md:151]: modified

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine

**Pull request merged**